functions/main.py

```diff
@@ -43,19 +43,13 @@ def get_list_from_api(title):
     }
     data_data['variables']['filter']['moreLikeThis']['like'] = title
     data = json.dumps(data_data)
-    # data = json.dumps({"query":data_data})
-    # data = json.load(data)
-    # print(str(data))
     try:
         r = requests.post("https://cofacts-api.g0v.tw/graphql", headers=headers, data=data)
         r.encoding = 'utf8'
         print(str(r))
-        # print(str(r.text))
-        soup = BeautifulSoup(r.text, 'html.parser')
-        # print(soup)
     except:
         print('\033[33m Error! \033[0m')
-        return 'no'
+        return 'error'
```

Comment:
please consider status code, and format. Use the same format otherwise it will be way too dynamics